app/helpers/file_set_helper.rb

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+module FileSetHelper
+  def display_title(fs)
+    fs.to_s
+  end
+
+  def present_terms(presenter, terms = :all, &block)
+    terms = presenter.terms if terms == :all
+    Sufia::PresenterRenderer.new(presenter, self).fields(terms, &block)
+  end
+
+  def display_multiple(value)
+    return if value.nil?
+    auto_link(html_escape(value.join(" | ")))
+  end
+
+  private
+
+  def render_visibility_badge(presenter)
+    # if can? :edit, presenter.solr_document
+    #   render_visibility_link presenter.solr_document
+    # else
```

Comment:
Is this code being commented out to remove it? or is it being left as a placeholder for some future work?